### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by mkane007**
Fixed issues:
1. OS detection fired `Exception` on Windows
2. using `mkdir` caused `Excetion` on Windows

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.1.3-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.1.4"
```
Comment (joprice) [inline version.sbt]: This version file is handled by the [sbt-release](https://github.com/sbt/sbt-release) plugin. It is automatically bumped when the release command is run, and should always be set to snapshot during normal development.

**Comment by joprice** [comment]
Thanks for the updates. I haven't used windows in years and don't have access to a windows machine/vm currently, so it's great to have someone checking cross-platform issues. The only issue is with the version change. Fix that and I'll merge and cut a release.

**Comment by mkane007** [comment]
@joprice done

**Comment by joprice** [comment]
@mkane007 just released 0.1.3.

**Pull request merged**